auth_ldaps/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+# Copyright (C) 2017 Creu Blanca
+# Copyright (C) 2018 Brainbean Apps
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "LDAPS authentication",
+    "version": "13.0.1.0.0",
+    "category": "Tools",
+    "website": "https://github.com/OCA/server-auth",
+    "author": "Brainbean Apps (https://brainbeanapps.com), "
+    "Creu Blanca, "
+    "Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "installable": True,
+    "application": False,
+    "summary": "Allows to use LDAP over SSL authentication",
+    "depends": ["auth_ldap"],
+    "external_dependencies": {"python": ["ldap"]},
```

Comment:
Runbot is failing because of this. I think you should change ldap for pyldap

Reply:
Indeed! Thanks for your feedback and your time @etobella